### DISCUSSION TITLE
Ability to play using mouse and keyboard on Android (for Samsung DeX desktop like experience)

### DISCUSSION THREAD

**Discussion by sztuk5** [ppy/osu#28524] [Ideas] [answered]
Hello, I think It would be awesome to add mouse and keyboard compatibility to Android version. I would love to have the ability to play in Samsung DeX like on normal PC :D

**Comment by Joehuu** [comment, marked as answer]
Should be possible next release (input library was changed to SDL, ref: https://github.com/ppy/osu/discussions/25138#discussioncomment-8193321). It was supposed to be in the latest release but was reverted due to some unrelated desktop platform problems with SDL3, it seems to be fixed now or at least downgraded to SDL2 for desktop platforms).
